packages/components/psammead-sitewide-links/CHANGELOG.md

```diff
@@ -1,57 +1,58 @@
+| 0.3.5   | [PR#424](https://github.com/bbc/psammead/pull/424) Add Snyk badge to readme                                                                                                                                  |
+| 0.3.4   | [PR#407](https://github.com/bbc/psammead/pull/407) Organise dependencies properly                                                                                                                            |
+| 0.3.3   | [PR#398](https://github.com/bbc/psammead/pull/398) Update dependencies. Using `rem` for media queries.                                                                                                       |
+| 0.3.2   | [PR#323](https://github.com/bbc/psammead/pull/323) Update storybook badge url                                                                                                                                |
+| 0.3.1   | [PR#347](https://github.com/bbc/psammead/pull/347) Constrain content width to Group 5 min width.                                                                                                             |
+| 0.3.0   | [PR#318](https://github.com/bbc/psammead/pull/318) Update to new font face and family.                                                                                                                       |
+| 0.2.1   | [PR#323](https://github.com/bbc/psammead/pull/323) Update readme storybook badge                                                                                                                             |
+| 0.2.0   | [PR#306](https://github.com/bbc/psammead/pull/306) Update background colour to Ebon & border colours to Shadow.                                                                                              |
+| 0.1.4   | [PR#264](https://github.com/bbc/psammead/pull/264) Resolving package-lock issues.                                                                                                                            |
+| 0.1.3   | [PR#245](https://github.com/bbc/psammead/pull/245) Ensures documentation consistent across component packages.                                                                                               |
+| 0.1.2   | [PR#231](https://github.com/bbc/psammead/pull/231) Add link to Storybook to README                                                                                                                           |
+| 0.1.1   | [PR#227](https://github.com/bbc/psammead/pull/227) Replace @bbc/gel-constants and @bbc/gel-foundations-styled-component with [@bbc/gel-foundations in Psammead](https://github.com/bbc/psammead/issues/226). |
+| 0.1.0   | [PR#154](https://github.com/bbc/psammead/pull/154) Create initial package with SitewideLinks component pulled in from [Simorgh](https://github.com/BBC-News/simorgh).                                        |
```

Comment:
looks like you messed up the formatting of the file

Reply:
Resolved.